cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -1,8 +1,10 @@
 {-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE StandaloneDeriving #-}
 
 {-# OPTIONS_GHC -Wno-noncanonical-monoid-instances #-}
+{-# OPTIONS_GHC -Wno-orphans            #-}
```

Comment:
Is the `{-# OPTIONS_GHC -Wno-orphans  #-}` pragma still necessary?

Reply:
Looks like we don't actually use the `Generic` instance.

Reply:
I created a PR to add these instances upstream.

Reply:
`SnapshotInterval` is missing (as you saw) two instances: `Generic` and `Eq`. I saw that @newhoggy already made a PR with needed changes thank you for that :) It is a change which we however can **NOT** use yet. 'Why??' one might wonder. Let me explain:
One important thing to understand is the fact that this PR is based on `ouroboros-consensus` merge commit `ec79b8725 Merge #2966` not the `master` branch. Why is that? Because the `master` has updated dependencies to `ledger-spec`, `cardano-prelude` and `cardano-base`. Those had breaking changes on the `cardano-node` and thus I've decided to update node's consensus dependency only up to relevant change (`ec79b8725 Merge #2966`).

What we can do about it:
1. I will make changes proposed by @Jimbo4350 and move the instances two `Cardano.Node.Orphans`
2. I will merge @newhoggy PR today to master(
3. Once node catches up with latest changes in consensus (that will probably happen during the Alonzo integration) we will be able to remove those orphans. Compiler will most likely tell us that this is needed.

Does that sounds like a plan?

Reply:
Sounds great!  Thanks!